### ISSUE TITLE
Wrong backbone name

### ISSUE THREAD

**Issue by Hackerman28** [minhhoangbui/lightweight_openpose_tensorflow#5]
Hi. Thanks for your TF2 implementation of lightweight openpose. I'm finding it very useful. While going over your [code](https://github.com/minhhoangbui/lightweight_openpose_tensorflow/blob/master/src/models/backbones.py) and the pytorch [implementation](https://github.com/Daniil-Osokin/lightweight-human-pose-estimation.pytorch/blob/master/models/with_mobilenet.py) of the same lightweight openpose model, I've noticed a discrepancy. For the mobilenet backbone, both have the same structure except padding on a single layer. But in the pytorch implementation, the repo author said it was a MobilenetV1 backbone but in your code, the backbone is named as Mobilenetv2. Is that a mistake or is the backbone different from the pytorch one?
I'm attaching screenshots of both the backbone implementations for reference.
![TF2](https://user-images.githubusercontent.com/36561958/105156391-c9142d80-5b31-11eb-8df7-c9b47cc31290.png)
![pytorch](https://user-images.githubusercontent.com/36561958/105156394-ca455a80-5b31-11eb-98af-8f7c10c59313.png)

**Comment by minhhoangbui** [comment]
@Hackerman28 Thank you for your feedback. It's my mistake and I will correct it asap. Can you tell me the layer where the padding is different?

**Comment by Hackerman28** [comment]
In pytorch implementation, the padding is given as 2 for layer 'mbn2_convdw7'. In your implementation, it is left as padding=same. That's the padding difference I found. Also I trained a couple of models with your implementation - mobilenet and resnet50. I am getting very poor mAP results when evaluating on COCO'17 validation dataset. Did you face this issue?

**Comment by minhhoangbui** [comment]
@Hackerman28 it's weird, I got AP@0.5 (which I think is mentioned in the paper) ~40.5, which is similar to that of paper. Furthermore, I got satisfactory result when inferencing, so I believe that my source code works.
In my opinion, ResNet is not suitable to OpenPose since we cannot get more than 2 layers if we want to keep original architecture
If you want better accuracy while maintaining acceptable speed, I suggest you dive into PifPaf. I have published my version of PifPaf in PyTorch, which is more understandable than the original one

**Comment by Hackerman28** [comment]
Can u provide your pretrained mobilenet model for my evaluation? I have followed your steps in the README.md and trained the mobilenet model for 60 epochs with 0.0006 LR. It would be helpful if u can provide me the train.yml file you used to train your best model because there were many versions of train.yml files in your commits so I may have used a wrong one.
Thanks in advance

**Comment by minhhoangbui** [comment]
``
COMMON:
  GPU: '0,1'
  retrain: True
  saved_dir: "/mnt/hdd10tb/Users/hoangbm/checkpoint/pose-estimation/"
  saved_epochs: 2
MODEL:
  name: 'lw_mobilenetv2'
  num_channels: 128
  num_stages: 1
  input_size: 368
  mobile: True
DATASET:
  name: 'coco'
  annotation_dir: '/mnt/hdd10tb/Users/hoangbm/datasets/coco2017'
  image_dir: '/mnt/ssd2/BACKUP/Datasets/COCO2017/images'
  stride: 8
  sigma: 7
  paf_thickness: 1
TRAIN:
  num_epochs: 40
  learning_rate: 0.001
  decay_step: 5000
  decay_rate: 0.9
  batch_size: 32
VAL:
  batch_size: 32
  ``
  From my point, you should use larger LR(0.001), sometimes I set smaller LR to finetune model since TF don't allow hooks for custom training in order to modify LR automatically.
